liquibase-standard/src/test/java/liquibase/resource/UrlDrivenResourceAccessorTest.java

```diff
@@ -0,0 +1,168 @@
+package liquibase.resource;
+
+import liquibase.ContextExpression;
+import liquibase.GlobalConfiguration;
+import liquibase.Labels;
+import liquibase.changelog.ChangeLogParameters;
+import liquibase.changelog.DatabaseChangeLog;
+import liquibase.exception.LiquibaseException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Objects;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class UrlDrivenResourceAccessorTest {
+
+    UrlDrivenResourceAccessor accessor;
+    DatabaseChangeLog changeLog;
+
+    @BeforeEach
+    public void setUp() {
+        this.accessor = new UrlDrivenResourceAccessor();
+        this.changeLog = new DatabaseChangeLog();
+        changeLog.setChangeLogParameters(new ChangeLogParameters());
+    }
+
+    @Test
+    public void rootChangeSet() throws LiquibaseException {
+        assertTrue(changeLog.include(
+                "classpath:liquibase/resource/changelog1.yml",
+                false,
+                true,
+                accessor,
+                new ContextExpression(),
+                new Labels(),
+                false,
+                DatabaseChangeLog.OnUnknownFileFormat.FAIL));
+
+        assertEquals(1, changeLog.getChangeSets().size());
+        assertEquals("cs1", changeLog.getChangeSets().get(0).getId());
+    }
+
+    @Test
+    public void nestedChangeSet_RelativePath_ThrowsByDefault() {
+        assertThrows(LiquibaseException.class, () -> changeLog.include(
+                "classpath:liquibase/resource/changelog2.yml",
+                false,
+                true,
+                accessor,
+                new ContextExpression(),
+                new Labels(),
+                false,
+                DatabaseChangeLog.OnUnknownFileFormat.FAIL));
+    }
+
+    @Test
+    public void nestedChangeSet_RelativePath_Allowed() throws LiquibaseException {
```

Comment:
Hi @andrus,

Build is currently failing because of this test that is failing with the below message:

```java
Error:  Errors: 
Error:    UrlDrivenResourceAccessorTest.nestedChangeSet_RelativePath_Allowed:72 � Liquibase Error parsing classpath:liquibase/resource/changelog2.yml : Illegal char <:> at index 9: classpath:liquibase/resource/subresources/child-changelog1.yml
```

As you can see on the execution details this test is only failing on Windows. Would it be possible for you to fix it?

Thanks,
Daniel.

Reply:
Yeah, I can reproduce the issue on Windows now. The culprit is [this code](https://github.com/liquibase/liquibase/blob/master/liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java#L983) :

```java
fileName = resourceAccessor.get(this.getPhysicalFilePath()).resolveSibling(fileName).getPath();
fileName = normalizePath(Paths.get(fileName).toString());
```

Surprisingly, `Paths.get("classpath:some/path)` works on MacOS, but (less surprisingly) fails on Windows with the error above. Trying to figure out if there's a non-invasive solution.

Reply:
Cool, thanks for taking the time to look at it.

Daniel.

Reply:
Ok, just pushed 19b7ac1d746428ceafe182efee3b32dce0bdac7a with a rather defensive fix ensuring that path normalization via `java.nio.file.Path` can work even for `classpath:` URLs.